docs/reference/modules/cluster/misc.asciidoc

```diff
@@ -22,6 +22,48 @@ user with access to the <<cluster-update-settings,cluster-update-settings>>
 API can make the cluster read-write again.
 
 
+[[cluster-shard-limit]]
+
+==== Cluster Shard Limit
+
+In a Elasticsearch 7.0 and later, there will be a soft cap on the number of
```

Comment:
Should we call it a "soft limit" to be in line with the terminology on similar settings elsewhere?

Reply:
Yes, I'll change all the instances of "cap" to "limit" - thanks!